azure_sdk_core/src/lib.rs

```diff
@@ -151,6 +151,21 @@ pub trait ClientRequestIdOption<'a> {
     }
 }
 
+pub trait AppendPositionSupport<'a> {
+    type O;
+    fn with_append_position(self, append_position: &'a str) -> Self::O;
+}
+
+pub trait AppendPositionOption<'a> {
+    fn append_position(&self) -> Option<&'a str>;
```

Comment:
If I understand correctly this is supposed to be a number. I think a `u32` would suffice and it would also get rid of the lifetime specifier! What do you think?

Reply:
Good call, made the swap.